pydantic/main.py

```diff
@@ -233,6 +241,7 @@ class BaseModel(metaclass=MetaModel):
         __fields__: Dict[str, Field] = {}
         __validators__: Dict[str, AnyCallable] = {}
         __config__: Type[BaseConfig] = BaseConfig
+        __root__: List[Any] = []
```

Comment:
I thought `__root__` doesn't have to be a list?

Reply:
Sorry, it's my mistake. I have changed ``List[Any]`` to ``Any``.